test/libyul/yulOptimizerTests/commonSwitchCasePrefixMover/assignToExpression.yul

```diff
@@ -0,0 +1,19 @@
+{
+  let x := calldataload(0)
+  switch x
+  case 0 {
```

Comment:
I think it would be best to keep the indents in Yul the same as in Solidity code (i.e. 4 spaces), at least in newly created files.